src/wp-includes/media.php

```diff
@@ -5774,7 +5829,7 @@ function wp_get_loading_optimization_attributes( $tag_name, $attr, $context ) {
 	 * If the element is in the viewport (`true`), potentially add
 	 * `fetchpriority` with a value of "high". Otherwise, i.e. if the element
 	 * is not not in the viewport (`false`) or it is unknown (`null`), add
-	 * `loading` with a value of "lazy".
+	 * `loading` with a value of "lazy" and `decoding` with a value of "async".
```

Comment:
This should be reverted. Images should always receive `decoding="async"`, not only when in the viewport.